cmake/modules/FindX10.cmake

```diff
@@ -0,0 +1,11 @@
+file(TO_CMAKE_PATH "${X10_TOOLCHAIN_PATH}" X10_CMAKE_TOOLCHAIN_PATH)
+
+find_file(X10_LIBRARY libx10.so ${X10_TOOLCHAIN_PATH} PATH_SUFFIXES usr/lib/swift/linux)
```

Comment:
Please don't look for `libx10.so`.  Additionally, I think that `find_library` is a much better approach than `find_file` for this case.